frank/geometry.py

```diff
@@ -22,10 +22,12 @@
    NOTE: The sign convention used here is xx.
 """
 
-from frank.constants import rad_to_arcsec, deg_to_rad
-
 import numpy as np
 from scipy.optimize import least_squares
+import logging
```

Comment:
Remove all logging from the main library in this PR 
```suggestion
```

Reply:
I'm just commenting it out for now, not reverting all the changes I've made that I think should be implemented eventually. I have thought about how to do the logging and think it important that some status reports are printed even when the code is run as a module, otherwise it will be less intuitive to use and harder to debug. I'll also comment the logging calls in the main library for now.

Reply:
I want them removed.